storybook/src/stories/screen/EmptyScreen.stories.svelte

```diff
@@ -0,0 +1,46 @@
+<script context="module" lang="ts">
```

Comment:
Same here, could it be possible to specify title, message, detail, ... in the Docs section?

Reply:
fixed